src/main/java/io/debezium/connector/db2/Db2Connector.java

```diff
@@ -74,10 +74,10 @@ protected void validateConnection(Map<String, ConfigValue> configValues, Configu
         ConfigValue hostnameValue = configValues.get(RelationalDatabaseConnectorConfig.HOSTNAME.name());
         // Try to connect to the database ...
         Db2ConnectorConfig connectorConfig = new Db2ConnectorConfig(config);
-        try (Db2Connection connection = new Db2Connection(connectorConfig.getConfig())) {
+        try (Db2Connection connection = new Db2Connection(connectorConfig.jdbcConfig())) {
```

Comment:
You should be able to just do `config.jdbcConfig()`, i.e. no need for the new method.

Reply:
Unfortunately, as I stated during our team call, the method is not part of ``RelationalDatabaseConnectorConfig```. Although I agree that it should probably be there. However, that would mean changes to pretty much all RDB connectors we have.

Personally, I have no problem moving it to the parent class. It's your call.